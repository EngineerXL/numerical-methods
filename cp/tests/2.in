8173192742620808189042833
9787734638775863246598114
