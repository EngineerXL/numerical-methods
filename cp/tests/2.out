79997041716341183067810512412503510113597083016962
